docs/cookbook/creating_landing_page_layouts_(enterprise).md

```diff
@@ -1,114 +1,93 @@
 # Creating Landing Page layouts (Enterprise)
 
-## Description
+A Landing Page has a customizable layout with multiple zones where you can place blocks with content.
 
-A Landing Page has a customizable layout with multiple zones where you can place predefined blocks with content.
-
-By default eZ Enterprise comes with a number of preset layouts. You can, however, add custom layouts with zones to your configuration.
-
-## Solution
-
-### Defining the layout
+A clean installation has only one default layout. You can preview more layouts in the Demo bundle.
 
 A Landing Page layout is composed of zones.
 
-#### Zone structure
+### Zone structure
 
 Each zone contains the following parameters:
 
 | Name             | Description                  |
 |------------------|------------------------------|
-| &lt;zone\_id&gt; | *Required*. A unique zone ID |
-| &lt;name&gt;     | *Required*. Zone name        |
-
-#### Defining a zone layout
-
-You can define a new layout file (e.g. in Twig) for a zone and include it in a Landing page layout.
-
-A Zone is a container for blocks. The best way to display blocks in the zone is to iterate over a blocks array and render the blocks in a loop.
+| `zone_id` | Required. A unique zone ID |
+| `name`     | Required. Zone name        |
 
-!!! note
-
-    For eZ Enterprise, the `data-studio-zone` attribute is required to allow dropping the Content into specific zones.
-
-``` html
-<!--Example zone.html.twig-->
-<div data-studio-zone="{{ zones[0].id }}">                                       
-    {# If a zone with [0] index contains any blocks #}
-    {% if zones[0].blocks %}                                                    
-        {# for each block #}
-        {% for block in zone[0].blocks %}                                               
-            {# create a new layer with appropriate id #}
-            <div class="landing-page__block block_{{ block.type }}">            
-                {# render the block by using the "ez_block:renderBlockAction" controller #}
-                {{ render_esi(controller('ez_block:renderBlockAction', {        
-                        {# id of the current content #}
-                        'contentId': contentInfo.id,                            
-                        {# id of the current block #}
-                        'blockId': block.id                                     
-                    }))
-                }}
-            </div>        
-        {% endfor %}    
-    {% endif %}
-</div>
-```
+## Layout configuration
 
-### Creating and configuring layouts
-
-In the Demo installation the layout configuration is stored in ezstudio-demo-bundle/Resources/config/default\_layouts.yml:
+The layout is configured in YAML files:
```

Comment:
We can't give a specific path, it depends on how the user has organized their config. It could be in `ezplatform.yml`, or could be a separate file called `layouts.yml`, or anything really.